src/api/graphql/resolvers/resolvers.js

```diff
@@ -1,3 +1,4 @@
 import userResolver from './userResolver';
```

Comment:
Should rename this file into index.js so that we only import `src/api/graphql/resolvers` instead `src/api/graphql/resolvers/resolvers`